src/assets/styles/layout.css

```diff
@@ -1,234 +0,0 @@
-article .hero > section,
-article .hero div > section,
-article .main > section,
-article .asides aside > section {
```

Comment:
.asides is not in use in publish, so I have not moved it into the styleguide https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-5b8e63d7964be0aaeb36dee44f1e1ca3R38 Looks like it came from beta.